src/pycram/worlds/bullet_world.py

```diff
@@ -222,14 +307,15 @@ def restore_physics_simulator_state(self, state_id):
     def remove_physics_simulator_state(self, state_id: int):
         p.removeState(state_id, physicsClientId=self.id)
 
-    def add_vis_axis(self, pose: Pose,
-                     length: Optional[float] = 0.2) -> None:
+    def _add_vis_axis(self, pose: Pose,
```

Comment:
yes add_vis_axis exists with the implementation in the World class, the implemented worlds only have to implement this internal method. this is because there is a wrapper around the methods that create or destroy objects because these methods change the state of the world in an irreversible way, as the restore state cannot work if there is a new object or if there is an object that got removed, so the original state of the world has to be updated, this could also be done using a decorator, but the problem the decorator in abstract classes are not enforced on the user so it may cause problems.